config/_default/menus/menus.en.yaml

```diff
@@ -1788,36 +1788,100 @@ main:
     pre: nav_security_monitoring
     identifier: security_platform
     weight: 160000
-  - name: Getting Started
-    url: security_platform/getting_started
+  - name: Security Monitoring
+    url: security_platform/security_monitoring/
     parent: security_platform
-    identifier: security_monitoring_getting_started
+    identifier: security_monitoring
     weight: 1
-  - name: Security Rules
-    url: security_platform/detection_rules/
+  - name: Getting Started
+    url: security_platform/security_monitoring/getting_started
+    parent: security_monitoring
+    identifier: security_monitoring_getting_started
+    weight: 101
+  - name: Security Home
+    url: security_platform/security_monitoring/security_home
+    parent: security_monitoring
+    identifier: security_monitoring_security_home
+    weight: 102
+  - name: Log Detection Rules
+    url: security_platform/security_monitoring/log_detection_rules
+    parent: security_monitoring
+    identifier: security_monitoring_log_detection_rules
+    weight: 103
+  - name: Default Rules
```

Comment:
when I click [here](https://docs-staging.datadoghq.com/sarina/new-section/security_platform/default_rules#cat-log-detection) when clicking down the nav, it messes up the left side toc and makes it disappear. I am not sure how much of a blocker this should be to merging, but worth talking to websites about to see how easy the fix is

Reply:
it might be that you can't use a `#` as a URL?

Reply:
@davidejones Hi there! Do you have any insight into this? It seems to happen here when you click on default rules, and in a couple other sections throughout the site. If a more long-term investigation is needed, let me know and I can create a card.

Reply:
I haven't confirmed but I think it might be because the default rules section isn't following the same path e.g everything in that subnav is `/security_platform/security_monitoring/something` but rules is `/security_platform/default_rules/`

Reply:
Do the other sections with this issue follow the same pattern above i mentioned?

Reply:
It was the anchor! @sarina-dd I just committed a fix to this branch https://github.com/DataDog/documentation/pull/10437/commits/099b3a4429897d86f44cae65a8a8db0362e3f380